core/src/main/java/com/dtolabs/rundeck/core/plugins/configuration/Description.java

```diff
@@ -76,4 +76,16 @@ public default <T extends PluginGroup> Class<T> getPluginGroupType(){
         return null;
     }
 
+    public default boolean isHighlighted() {
+        return false;
+    }
+
+    public default int getOrder() {
+        return 0;
+    }
+
+    public default boolean isBuiltInPlugin() {
```

Comment:
comment: I'm not sure about adding this boolean to the Description, for only the temporary usage for Job Reference.  Is there a reason to distinguish between Job reference and other Highlighted plugins such as Command step?

Reply:
The way that builtin plugin is built in the gsp page is different than true plugins. Before, the builtin plugins was hard coded in the gsp, but now it is built from a list of plugins that define the order of the plugins in the page. This method help job reference to be built as before. https://github.com/rundeck/rundeck/blob/8418a3bdedf0f28699363bfe89d74414f28ed0f5/rundeckapp/grails-app/views/execution/_wfAddStep.gsp#L82

Reply:
hmm, maybe it can just check the provider+service name? I'd prefer to avoid adding the `isBuiltInPlugin` to such a core type used by all plugins, if it is just for one purpose